src/data/defaults.txt

```diff
@@ -1774,6 +1774,7 @@ user	_aug30Cast	false
 user	_aug31Cast	false
 user	_augSkillsCast	0
 user	_augTodayCast	false
+user	_authorityUsed	0
```

Comment:
I would rather this ended in `Cast` than `Used`, which typically (but admittedly not always) differentiates betwen skills and items in pref tracking

Reply:
I followed Envy as a model, but I'll change it Monday.

Reply:
looks like about a quarter of "Used" are skills. "Uses" is another: 16 skills, 9 items.

if it wants to move to Cast, I'd rather have "_assertAuthorityCast".

Reply:
It's not universal yeah. Sadly the pref system is quite user-hostile to change down the line.

Reply:
Updated the pref from my phone.